From bf29ff6d667efd7ac9c9afab70fca2c470765f09 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 7 Sep 2018 09:25:53 +0200
Subject: [PATCH 0700/1051] arm64: dts: marvell armada-3720-db-B: add comphy
 reference to sgmii port

Adding phy description to sgmii will allow neta driver to configure comphy
via comphy-a3700 driver.

Change-Id: I0d67ddfd203321f9dd06a0b15cef7b0e03f1789b
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db-B.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
index 1f9e95bdb656..bcfae3f2054d 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
@@ -10,4 +10,6 @@
  */
 &eth1 {
 	status = "okay";
+	/* Generic PHY, providing serdes lanes */
+	phys = <&a3700_comphy0 1>;
 };
-- 
2.17.1

